src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+@import './variables';
+@import './stars';
+
+.card__link {
+  box-sizing: border-box;
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 40px;
+  border-radius: 5px;
+  background: $blue-accent;
+  border: none;
+
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-transform: uppercase;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $white;
+  padding: 16px;
+  transition: all 0.3s ease-in;
```

Comment:
it's not a good practice to use `all`, specify what exactly you'd like to change